code/style.css

```diff
@@ -1,3 +1,257 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  -webkit-font-smoothing: antialiased;
+  text-rendering: optimizeLegibility;
+}
+
 body {
-  background: #FFECE9;
+  font-family: 'PT Sans', sans-serif;
+}
+
+a {
+  text-decoration: none;
+  font-weight: 600;
+}
+
+.user-container {
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+}
+
+.github-tracker {
+  margin-top: 3.5em;
+  padding: 2em;
+  font-size: 30px;
+  font-weight: 400;
+  letter-spacing: .1em;
+  align-self: center;
+}
+
+.user-header {
+  display: none;
+}
+
+.github-user {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+}
+
+.github-logo {
+  background-color: white;
+  border-radius: 1em;
+  height: 1.7em;
+}
+
+.user-name {
+  padding-left: .5em;
+  font-weight: 600;
+  font-size: 20px;
+  color: black;
+}
+
+.user-img {
+  height: 11em;
+  align-self: center;
+  border-radius: 8em;
+  margin-bottom: 3.5em;
+}
+
+.projects-container {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: nowrap;
+  align-items: center;
+}
+
+.projects-header {
+  margin-bottom: 2em;
+  font-size: 28px;
+  letter-spacing: .5em;
+  font-weight: 100;
+  text-align: center;
+}
+
+.project-name {
+  font-size: 25px;
+  color: white;
+  cursor: pointer;
+  padding: .8em .5em;
+  margin-top: 1em;
+  width: 90vw;
+  border: none;
+  outline: none;
+  transition: 1s;
+  overflow: visible;
+  text-align: center;
+  background-color: rgb(104, 131, 180);
+}
+
+.project-name:hover {
+  font-weight: 800;
+  background-color: rgb(59, 83, 129);
+}
+
+.panel {
+  max-height: 0;
+  overflow: hidden;
+  transition: max-heigth 1s ease-out;
+  font-size: 20px;
+  transition: max-heigth 1s ease-out;
+  color: rgb(104, 131, 180);
+  text-align: center;
+  margin-bottom: 1em;
+}
+
+.panel a {
+  color: rgb(104, 131, 180);
+}
+
+.active + .panel {
+  display: block;
+  transition: 1s;
+  margin: 1em 0;
+}
+
+.active {
+  font-weight: bold;
+  border-radius: 1em;
+  background-color: rgb(59, 83, 129);
+}
+
+.chart-header {
+  margin: 2.5em 0 2em 0;
+  font-size: 28px;
+  letter-spacing: .5em;
+  font-weight: 100;
+  text-align: center;
+}
+
+.chart {
+  height: 300px;
+  width: 300px;
+  margin-left: 2.5em;
+}
+
+.chart-container {
+  display: flex;
+  justify-content: center;
+}
+
+footer {
+  height: 15vh;
+  width: 100vw;
+  text-align: center;
+  padding-top: 4em;
+  margin-top: 4em;
+}
+
+
+/* TABLET */
+@media screen and (min-width: 667px) and (max-width: 1024px) {
```

Comment:
I think it's enough with the min-width in tablet mode, cause your desktop starts with the min-width 1024 anyway, just a thought